pkgs/development/python-modules/protobuf/default.nix

```diff
@@ -1,5 +1,6 @@
 { buildPackages
 , lib
+, buildPackagesProtobuf ? buildPackages.protobuf
```

Comment:
```suggestion
, buildPackagesProtobuf
```
Please move this to top-level.

Reply:
this was removed, instead using version as a way to find the correct package